pages/index.js

```diff
@@ -14,9 +14,13 @@ export default function HomePage() {
         <ListLayout>
           {bill.map((item) => (
             <ItemLayout key={item.id}>
-              <TitleText>{item.title}</TitleText>
-              <Link href={`/bills/${item.id}`}>View Details</Link>
-
+              <TitleText>
+                {item.title}
+                <br />
+                <DetailsLink href={`/bills/${item.id}`}>
+                  View Details
+                </DetailsLink>
+              </TitleText>
```

Comment:
Do you really need to nest the link? 🤔

Reply:
i did it temporarily so the link is underneath the title and not in the middle of the list item. will also change it with styling but i have it marked to not nest it again ^_^